### ISSUE TITLE
Slicing fails in 5.7.1 and 5.8

### ISSUE THREAD

**Issue by danbi** [Ultimaker/Cura#19523]
### Cura Version

5.7.1 and 5.8.0

### Operating System

MacOS Sonoma

### Printer

Voron 350 with 0.6mm nozzle

### Name abnormal settings

Tried all standard presets

### Describe model location

as loaded, sits on the build-late

### Describe your model

It works fine in Cura 5.6

### Add your .zip here ⬇️

[sand-spoon.stl.zip](https://github.com/user-attachments/files/16573739/sand-spoon.stl.zip)

**Comment by GregValiant** [comment]
Thanks for the report.
I didn't have any trouble slicing it with my settings and a 0.6 nozzle.
You will need to attach your Cura.log file here and maybe a project file as well.
![image](https://github.com/user-attachments/assets/a61d311e-7b32-4d9d-b60e-fd06c05d529f)

**Comment by danbi** [comment]
It must be some corruption of the printer configurations then, as I was able to successfully slice for an Sunlu S9. Maybe also attach the printer configs too?

**Comment by MrKrafter3D** [comment]
someone has the problem that the program crash when you´re programming the color chance with pause at height? Cura V5.8.0

**Comment by danbi** [comment]
Here is Cura.log, from 5.8.0. Also attaching a project file with this model.

[cura-5.8-logs-profile.zip](https://github.com/user-attachments/files/16578298/cura-5.8-logs-profile.zip)

**Comment by GregValiant** [comment]
If you are not using "USB Printing" then go to the MarketPlace, click on the "gear" icon, and scroll down and disable the "USB Printing" plugin.  If you have USB devices plugged in to the computer, and if those devices are configured as serial devices, then USB Printing will keep trying to connect even though it might be a pair of headphones rather than a printer.  That can cause problems and there are errors in the log that appear to be caused by it.

But there are a lot of other error descriptions in the log. 
You will need a UM person ( @HellAholic ?) to check that log file.  There are so many errors described in it that I get lost.  
The project itself sliced for me with no issues.

**Comment by danbi** [comment]
On this Mac I have three different versions of Cura (5.6.0, 5.7.1 and 5.8.0). Only 5.6.0 works. They have different working directories etc. This is Intel based Mac.

On another Mac I have much more: 4, 5.0, 5.1, 5.2.1, 5.3.1, 5.4, 5.5, 5.6, 5.8. All work. This is M1 based Mac.

It may be, that 5.7 did something wrong during upgrading the config directory which propagated further. I may try to completely remove any Cura remnants from the computer which will likely resolve it, but it is important to understand what the regression is. I vaguely remember there was some more radical change wrt Vorons in 5.7. The 5.7 was never installed on the second Mac.

Already did experiment adding new Voron2 350 printer, no improvement.

**Comment by danbi** [comment]
I have experimented with this:

Disabled OctoPrint (3.7.3) for the printer, no help.
Deinstalled OctoPrint, no help.
Disabled USB Printing, no help.
Moved all thee instances of Cura from the Application folder, installed again Cura 5.8, no help.
Moved away the ~/Library/Application Support/cura directory, then started Cura 5.8.0 again and everything works.

So obviously there is some junk in that directory, most likely happened during the use of Cura 5.7.0 as any errors seen are since then.
Experimented by moving back the old cura directory, which had three settings directories for 5.6, 5.7 and 5.8, removed 5.7 and 5.8 but sill the same error. Which means there is something in my configuration, which does not bother Cura 5.6 (which works) but is not properly migrated to 5.7 or 5.8.

I'm attaching my ~/Library/Application Support/cura directory for reference.

[cura.zip](https://github.com/user-attachments/files/16739396/cura.zip)

**Comment by danbi** [comment]
Eventually found the reason why this is happening. I had introduced new fdmextruder.def.json with the Sun S9 profile, which was apparently different from the one in Cura and in particular did not define machine_extruder_start_code_duration. The error about this was in cura.log. Instead of adding the definition from the Cura software I just deleted the file, which might have been outdated anyway, presumably the bundled one is used. 

Everything works so far, possibly Sunlu S9 should be investigated better.

Perhaps Cura 5.6 did not have/use this parameter and this is why it had no issues.
